Review user guide (#979)
ansible/receptor: docs/source/installation.rst

Download and extract precompiled binary for your OS and platform from `the releases page on GitHub <https://github.com/ansible/receptor/releases>`_

> Alternatively, Receptor can be compiled from source code (Golang 1.20+ required)

Reviewer: ```suggestion
Alternatively, you can compile Receptor from source code (Golang 1.20+ required)
```

Alternatively, you can compile from source code (Golang 1.20+ required)

.. code-block:: bash

    make receptor

> Test the installation with

Reviewer 2: ```suggestion
Test the installation with the following commands:
```

Author: better, thanks!

.. code-block:: bash

    receptor --help
    receptor --version

> The preferred way to interact with receptor nodes is to use the receptorctl command line tool

Reviewer 2: ```suggestion
The preferred way to interact with receptor nodes is to use the ``receptorctl`` command line tool
```

Reviewer 2: Should "receptor" start with a capital R since it is the name?

Author: added both suggestions
